Applications/JupyterHub.qmd

```diff
@@ -108,6 +116,36 @@ Working with a Notebook is pretty convenient and the supporting text should enab
 | M                 | Change cell to Markdown (text) mode |
 
 
+## Storage
+
+When you start the notebook, in the file navigation pane (the sidebar - #2) you will see three folders:
+
+* `/mystorage`
```

Comment:
```suggestion
* `~/mystorage`
```

If you write it like `/foo`, it looks like a toplevel folder. I would add a `~` prefix (to indicate that it is under one's home folder),  or without anything, so just `mystorage`

Reply:
Thanks for the suggestion @soxofaan  - I think without any symbol would be the best solution as this is the way it is shown in the UI